Software/main/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2018"
 [dependencies]
 state-machine = { path = "../state-machine" }
 data-acquisition = { path = "../data-acquisition" }
+nova-software-common = { path = "../../../nova-software-common" }
```

Comment:
We'll probably want to change this to a git dependency later once the structs stabilize